ImpeccableHQ/swapr-dapp
# feature: limit order on pro mode

# Summary

Fixes #1670

Merge limit orders into pro mode

![image](https://user-images.githubusercontent.com/23079677/211548444-5e6d5c23-739c-4a49-97c0-5d45b92c3e21.png)

  # To Test

1. Open swap in pro mode
2. Click in limit orders tab
- [ ] Limit orders should be available within advanced trading mode
